SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -310,6 +311,7 @@ Interest InterestFactory::CreateInterest() const
 	if (Settings->bEnableServerQBI)
 	{
 		// If we have server QBI, every actor needs a query for the server
+		// TODO(jacques): Use worker interest instead (UNR-2656)
 		AddActorInterest(ResultInterest);
```

Comment:
both AddActorInterest and AddServerSelfInterest add a component query to the Position component. Is that allowed?

Reply:
yep! Totally fine. Interest component is a map from component ID to list of queries, so I wrote a helper which just either creates the entry or appends it to the list when you add a new query to a component.